llvm/lib/IR/RepoTicket.cpp

```diff
@@ -118,76 +127,122 @@ static GODigestState accumulateGOFinalDigestOrInitialDigestAndGetGODigestState(
 
 // Update the GO's hash value by adding the hash of its dependents.
 template <typename Function>
-static bool updateDigestUseDependenciesAndContributions(
-    const GlobalObject *GO, MD5 &GOHash, GOStateMap &Visited, GOInfoMap &GOIMap,
-    Function AccumulateGODigest) {
+static auto updateDigestUseDependenciesAndContributions(
+    const GlobalObject *GO, GOStateMap &Visited, GOInfoMap &GOIMap,
+    MemoizedHashes &HashTable, Function AccumulateGODigest)
```

Comment:
I'm not keen on the variable name "HashTable". It sounds like a data type. Maybe GOHashCache or something like that would be better.

Reply:
Renamed, thanks.